esi_leap/tests/db/sqlalchemy/test_api.py

```diff
@@ -259,4 +266,122 @@ def test_offer_get_by_name(self):
 
     def test_offer_get_by_name_not_found(self):
 
-        assert api.offer_get_by_uuid('some_uuid') is None
+        self.assertEqual(api.offer_get_by_name('some_name'), [])
+
+    def test_offer_destroy(self):
+
+        offer = api.offer_create(test_offer_2)
+        api.offer_destroy(offer.uuid)
+        self.assertEqual(api.offer_get_by_uuid('offer_2'), None)
+
+    def test_offer_destroy_not_found(self):
+
+        self.assertEqual(api.offer_get_by_uuid('offer_4'), None)
+
+    def test_offer_update(self):
+
+        offer = api.offer_create(test_offer_3)
+        o = api.offer_create(test_offer_2)
+        values = {'start_time': o.start_time,
+                  'end_time': o.end_time}
+        api.offer_update(offer.uuid, values)
+        res = api.offer_get_by_uuid(offer.uuid)
+        self.assertEqual(res.start_time, offer.start_time)
+        self.assertEqual(res.end_time, offer.end_time)
+
+    def test_offer_get_all(self):
+
+        o = api.offer_create(test_offer_2)
+        offer = api.offer_create(test_offer_3)
```

Comment:
I would use variable names like `o1` and `o2`; if the objects are similar, use similar variable names

Reply:
makes sense